src/services/CertificateGenerationService.ts

```diff
@@ -108,6 +132,9 @@ class CertificateGenerationService {
         body: payload,
       }),
     };
+
+    // TODO remove this log after testing
```

Comment:
Can this be removed now?

Reply:
Do we have a ticket in the backlog for that merge, wondering if it worth raising this somewhere as a reminder for us to take this and the other debug log out

Reply:
There is a ticket to merge the holding branch into develop, so will add it as an action on that ticket.